src/utils/handleLatestFeed.ts

```diff
@@ -0,0 +1,40 @@
+import type { InterfaceQueryOrganizationPostListItem } from './interfaces';
+import type { InterfaceQueryOrganizationEventListItem } from './interfaces';
+
+function sortandSliceEventsByStartDate(
```

Comment:
```suggestion
function sortAndSliceEventsByStartDate(
```

Reply:
> I guess there's a typo in the variable name here.

Yea, I made it more consistent as it was used in other places